src/main/java/it/cosenonjaviste/security/jwt/utils/JwtTokenBuilder.java

```diff
@@ -210,7 +215,7 @@ public JwtTokenBuilder expirySecs(int seconds) {
 	 * @return {@link JwtTokenBuilder}
 	 */
 	public JwtTokenBuilder algorithm(Algorithm algorithm) {
-		options.setAlgorithm(algorithm);
+		this.algorithm = algorithm;
```

Comment:
Yes I kept the method to prevent breaking changes. Actually I just noticed that I didn't create an overload method `create(Algorithm)` (as you propose) in Builder class, I created only for Verifier. I added in the new commit of the pull request